lib/config/presets/internal/replacements.ts

```diff
@@ -91,6 +93,32 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'react-query-devtools-to-scoped': {
+    description:
+      '`react-query/devtools` became scoped under the `tanstack` organization.',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=2.6.3 <4.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['react-query/devtools'],
+        replacementName: '@tanstack/react-query-devtools',
+        replacementVersion: '4.0.0',
+      },
+    ],
+  },
+  'react-query-to-scoped': {
+    description:
+      '`react-query` became scoped under the `tanstack` organization.',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=3.39.2 <4.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['react-query'],
+        replacementName: '@tanstack/react-query',
+        replacementVersion: '4.0.5',
```

Comment:
The first `v4` version of the scoped `@tanstack/react-query` package is `4.0.5`, instead of the expected `4.0.0`... 😄 

Check out the `versions` tab on npm: https://www.npmjs.com/package/@tanstack/react-query